.github/workflows/ci.yml

```diff
@@ -392,6 +392,13 @@ jobs:
     env:
       GOTEST: gotestsum --
     steps:
+      - name: Fix up for actuated
+        # https://github.com/containerd/containerd/pull/9920#issuecomment-2024823587
+        if: matrix.os == 'actuated-arm64-4cpu-16gb'
+        run: |
+          sudo chmod g+rx /home/runner /home/runner/actions-runner /home/runner/actions-runner/_work
+          sudo chmod o+rx /home/runner /home/runner/actions-runner /home/runner/actions-runner/_work
```

Comment:
Added these chmods (thanks @rata), but the userns test is still failing

Reply:
@AkihiroSuda sorry for the slow response, I'm traveling and have limited internet access some days.

I see this is only on crun now, right? I thought it was on runc before and crun was working. Am I misremembering or was this the case before too?

From this error and statx manpage:
```
  Mar 29 18:13:27.618: INFO: Unexpected error occurred: rpc error: code = Unknown desc = failed to start sandbox "2025bef91c8629878153eea18d9c0f008277a90218f54c7811e1ad5812e448c3": failed to create containerd task: failed to create shim task: OCI runtime create failed: cannot stat `/home/runner/actions-runner/_work/containerd/containerd/tmp.oHgXPVzq1R/state/io.containerd.grpc.v1.cri/sandboxes/2025bef91c8629878153eea18d9c0f008277a90218f54c7811e1ad5812e448c3/shm`: Permission denied: unknown
```

I think some path there is not `o+rx`. And my bet is that `tmp.oHgXPVzq1R` is the one that hasn't it. Are you sure you checked all of them? That path seems like tmp dir created by go, that usually only has permissions for the owner (go creates them this way, we have some logic in runc tests to fix that in one way or another).

cc @giuseppe @saschagrunert in case they have any insight. @saschagrunert when writing the tests (https://github.com/kubernetes-sigs/cri-tools/pull/1354) did you tried with containerd and crun? Did you have to make any adjustments to solve this issue (cannot stat `.../shm`: permissions denied)?

Reply:
@rata we have a containerd crun test suite for various release branches in https://github.com/kubernetes-sigs/cri-tools/actions/runs/8448172906/job/23139752351

Reply:
@saschagrunert thanks!

My best bet is this is the trick: https://github.com/kubernetes-sigs/cri-tools/pull/1354/files#diff-a1e357914bebffbac9aaaed1c4768a71ad4dec62d56a78f7759b68fe8df4c5cbL302-R303 . @AkihiroSuda can you check that?

Reply:
After add o+rx on temp dir, it's still failed

[FAILED] failed to start container: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim task: OCI runtime create failed: mount_setattr `/etc/hostname`: Function not implemented: unknown

https://github.com/containerd/containerd/actions/runs/8844542494/job/24286639305?pr=10137

I think we should skip that case in ARM platform when we use crun.

Reply:
Oh, if that is the error now, yes, we should skip it. Not just for crun, as runc will fail in the same way when there is a release that supports idmap mounts (hopefully in the next few months).

@fuweid thanks!